### ISSUE TITLE
Problem with long app/file names

### ISSUE THREAD

**Issue by ChrisBlankDe** [SimonOfHH/D365BCAppHelper#16]
We are running in some issues with long app/file names.
```
AUSFÜHRLICH: Load Manifest from App File 'C:\temp\rtps\Any Publisher_App with long name which really exists Test_20.0.41580.0.runtime-20.6.47024.0-de.app'
AUSFÜHRLICH: Getting information from C:\temp\rtps\Any Publisher_App with long name which really exists Test_20.0.41580.0.runtime-20.6.47024.0-de.app
AUSFÜHRLICH: Copying file C:\temp\rtps\Any Publisher_App with long name which really exists Test_20.0.41580.0.runtime-20.6.47024.0-de.app to temporary directory
AUSFÜHRLICH: Creating temporary path C:\Users\ADMINI~1\AppData\Local\Temp\238b631c-25e4-469c-90d5-12bacb96cd86\Any Publisher_App with long name which really exists Test_20.0.41580.0.runtime-20.6.47024.0-de
AUSFÜHRLICH: Copying C:\Users\ADMINI~1\AppData\Local\Temp\238b631c-25e4-469c-90d5-12bacb96cd86\Any Publisher_App with long name which really exists Test_20.0.41580.0.runtime-20.6.47024.0-de\Any Publisher_App with long name which really exists Test_20.0.41580.0.runtime-20.6.47024.0-de.app to temporary path C:\Users\ADMINI~1\AppData\Local\Temp\238b631c-25e4-469c-90d5-12bacb96cd86\Any Publisher_App with long name which really exists Test_20.0.41580.0.runtime-20.6.47024.0-de (as C:\Users\ADMINI~1\AppData\Local\Temp\238b631c-25e4-469c-90d5-12bacb96cd86\Any Publisher_App with long name which really exists Test_20.0.41580.0.runtime-20.6.47024.0-de\Any Publisher_App with long name which really exists Test_20.0.41580.0.runtime-20.6.47024.0-de.app)
Copy-Item : Der angegebene Pfad und/oder Dateiname ist zu lang. Der vollständig qualifizierte Dateiname muss kürzer als 260 Zeichen und der Pfadname kürzer als 248 Zeichen sein.
In C:\Program Files\WindowsPowerShell\Modules\D365BCAppHelper\0.0.0.25\Export\Private\Get-D365BCManifestFromAppFile.ps1:89 Zeichen:13
+             Copy-Item $Filename -Destination $NewFilename | Out-Null
+             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Copy-Item], PathTooLongException
    + FullyQualifiedErrorId : System.IO.PathTooLongException,Microsoft.PowerShell.Commands.CopyItemCommand
 
```
Maybe we can avoid creating a folder named like the app file which is copied into it.

https://github.com/SimonOfHH/D365BCAppHelper/blob/f19ee72fc56653fa1fd11197efbfe9a542819d30/Export/Private/Get-D365BCManifestFromAppFile.ps1#L89

**Comment by SimonOfHH** [comment]
Hi @ChrisBlankDe ,
thanks for reporting this 😊 I guess you're right. Let me test this myself and get back to you. I'll try to investigate shortly.

Cheers
Simon

P.S.: "_Any Publisher_App with long name which really exists Test_20.0.41580.0.runtime-20.6.47024.0-de.app_" is an awesome filename 😅

**Comment by SimonOfHH** [comment]
Hi again @ChrisBlankDe ,

just had a moment to check this out. I was unable to reproduce the probleme (since obviously my setup already allows more than 260 chars) but I implemented a change anyway and (hopefully) fixed it (with commit 3e7d82042f9cfa3b9a31bb44c8c1e8cb0604e189). 
Will close this issue now, if you notice anything else regarding this issue feel free to reopen.
